imageprocessor/imageprocessor-core/src/main/java/pl/edu/uj/kimage/Image.java

```diff
@@ -0,0 +1,5 @@
+package pl.edu.uj.kimage;
+
+public class Image {
+    //TODO this is not finished
```

Comment:
TODO should include information about what is missing or why you cannot implement this code now. It's obvious that it is not finished :lollipop: